spt/utils/patterncontainer.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <map>
+#include <string>
+#include <vector>
+#include "SPTLib\memutils.hpp"
+
+#define PVOID void*
```

Comment:
```suggestion
using PVOID = void*;
```

Reply:
I'm too late xd but in general is it really better to typedef this vs just using `void *`?

Reply:
I thought about commenting about this but then I was like eh